hack/dep-update.sh

```diff
@@ -23,26 +23,26 @@ done
     echo $_sync_only
     cd staging/src/kubevirt.io/api
     if [ "${_sync_only}" == "false" ]; then go get $@ ./...; fi
-    go mod tidy
+    go mod tidy -compat=1.17
```

Comment:
Last time around @mhenriks suggested that when we move to go 1.18, we can remove -compat=1.17 so maybe comment that

Reply:
Added